devices/common_patches/h7_common_bootflash.yaml

```diff
@@ -0,0 +1,141 @@
+# Common SVD errors for stm32h7r/s
+
+_derive:
+  LPTIM2: LPTIM1
+  LPTIM3: LPTIM1
+  LPTIM5: LPTIM4
+
+  TIM3: TIM2
+  TIM4: TIM2
```

Comment:
Are you sure?
TIM2 and TIM5 usually are 32-bit.
When TIM3 and TIM4 are 16-bit timers.

Reply:
Ok. RM says all are 32-bit. Interesting.